src/NuGet.LibraryModel/LibraryRange.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using NuGet.Versioning;
+
+namespace NuGet.LibraryModel
```

Comment:
Lol. I literally made the same change at one point while working on this but was like "Nah. I bet there's a reason it's a bool"